.github/workflows/build.yaml

```diff
@@ -0,0 +1,19 @@
+name: ClickHouse Kafka Connect Tests CI
+
+on: [push]
```

Comment:
Shouldn't we run tests on pull requests against `main`?

Reply:
I want to run integration test when we have PR against main

Reply:
I want to run integration test against main when we are doing PR, but i need first verify that github actions can handle it (memory & CPU consumption since it bootstrap a full env)